src/backend/ai-reviewer-api/pom.xml

```diff
@@ -223,7 +223,7 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 				<configuration>
-					<mainClass>ca.bc.gov.open.jag.aireviewerapi.EfilingReviewerApiApplication</mainClass>
+					<mainClass>ca.bc.gov.open.jag.aireviewerapi.AIReviewerApiApplication</mainClass>
```

Comment:
Should 'AIReviewerApiApplication' be 'AiReviewerApiApplication'? (i.e. lower case 'i')

Reply:
yeah, that's good. Updated name to AiReviewerApiApplication